endpoints/get.go

```diff
@@ -63,12 +63,55 @@ func writeGetResponse(w http.ResponseWriter, id string, value string) (error, in
 		w.Header().Set("Content-Type", "application/json")
 		w.Write([]byte(value)[len(backends.JSON_PREFIX):])
 	} else {
-		return fmt.Errorf("uuid=%s: Cache data was corrupted. Cannot determine type.", id), http.StatusInternalServerError
+		return fmt.Errorf("Cache data was corrupted. Cannot determine type."), http.StatusInternalServerError
 	}
 	return nil, http.StatusOK
 }
 
-func respondAndLogError(w http.ResponseWriter, err error, status int) {
-	log.Errorf(err.Error())
-	http.Error(w, err.Error(), status)
+// Will prefix error messages with "GET /cache" and, if uuid string list is passed, will
+// follow with the first element of it in the following fashion: "uuid=FIRST_ELEMENT_ON_UUID_PARAM".
+// Expects non-nil error
+func handleException(w http.ResponseWriter, err error, status int, uuid string) {
+	// Build message
+	msg := "GET /cache"
+	if len(uuid) > 0 {
+		msg = fmt.Sprintf("%s uuid=%s:", msg, uuid)
+	}
+	msg = fmt.Sprintf("%s %s", msg, err.Error())
```

Comment:
Consider simplifying to:
```
msg = fmt.Sprintf("GET /cache uuid=%s: %s", id, err.Error())
```

I personally think a `GET /cache uuid=: key not found` would be alright., or else perhaps:

```
if len(uuid) > 0 {
  msg = fmt.Sprintf("GET /cache uuid=%s: %s", id, err.Error())
} else {
  msg = fmt.Sprintf("GET /cache: %s", err.Error())
}
```

Both will only create 1 string instead of the 3 in the code we have now.

Reply:
In practice missing keys won't get a `GET /cache uuid=: Key not found` error but a `GET /cache Missing required parameter uuid" because they'll get thrown in line 20. The `if/else` statement looks pretty neat. Done